src/ms3/bs4_parser.py

```diff
@@ -2220,6 +2229,46 @@ def get_instrument_name(self, staff_name: Union[str, int]):
         else:
             return fields_data[staff_name]['trackName']
 
+    def add_versions(self, new_values, version):
+        """
+        Adds version in the format I, II etc
+        Args:
+            new_values: the distionary of fields to update
```

Comment:
typo

Reply:
Changed it in [a0f79de](https://github.com/johentsch/ms3/pull/84/commits/a0f79decb44ae71d612255c6472b8c3a37e50b21)